modules/infinite_odoscope/js/models/odoscope-article-model.js

```diff
@@ -4,7 +4,8 @@
 
   window.OdoscopeArticleModel = BurdaInfinite.models.base.BaseModel.extend({
     defaults: {
-      loadingIndex: 0
+      loadingIndex: 0,
+      currentURL: null,
```

Comment:
Why is url `null`ified?  Isn't it supposed to be a string?

Reply:
now empty string